modules/ting/ting.module

```diff
@@ -181,20 +168,6 @@ function ting_menu_alter(&$items) {
   }
 }
 
-/**
- * Implements hook_ding_install_tasks().
- */
-function ting_ding_install_tasks() {
```

Comment:
Has a similarly function be created for setting up and selection an search provider in the install profile?

Reply:
No. Core only ships with a single usable search provider. It is enabled by default.

Reply:
👍